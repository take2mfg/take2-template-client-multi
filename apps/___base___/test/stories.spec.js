import path from 'path';
import initStoryshots from '@storybook/addon-storyshots';
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';

let storybookUrl = 'file:///';
storybookUrl += path.resolve(__dirname, '../../../.storybook-static');

initStoryshots({
  suite: 'Image storyshots',
  test: imageSnapshot({ storybookUrl }),
});
