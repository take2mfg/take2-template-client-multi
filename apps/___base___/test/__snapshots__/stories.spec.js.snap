// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots BaseApp|Button with text 1`] = `
<div>
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <button
        onClick={[Function]}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "border": "1px solid #eee",
            "borderRadius": 3,
            "cursor": "pointer",
            "fontSize": 15,
            "margin": 10,
            "padding": "3px 10px",
          }
        }
        type="button"
      >
        Hello Button
      </button>
    </div>
  </div>
  <button
    onClick={[Function]}
    style={
      Object {
        "background": "#28c",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>
  <div
    style={
      Object {
        "background": "white",
        "bottom": 0,
        "display": "none",
        "left": 0,
        "overflow": "auto",
        "padding": "0 40px",
        "position": "fixed",
        "right": 0,
        "top": 0,
        "zIndex": 99999,
      }
    }
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "#28c",
          "border": "none",
          "borderRadius": "0 0 0 5px",
          "color": "#fff",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "sans-serif",
          "fontSize": "12px",
          "padding": "5px 15px",
          "position": "fixed",
          "right": 0,
          "top": 0,
        }
      }
      type="button"
    >
      ×
    </button>
    <div>
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "backgroundColor": "#fff",
            "border": "1px solid #eee",
            "borderRadius": "2px",
            "color": "#444",
            "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
            "marginBottom": "20px",
            "marginTop": "20px",
            "padding": "20px 40px 40px",
          }
        }
      >
        <div
          style={
            Object {
              "borderBottom": "1px solid #eee",
              "marginBottom": 10,
              "paddingTop": 10,
            }
          }
        >
          <h1
            style={
              Object {
                "fontSize": "35px",
                "margin": 0,
                "padding": 0,
              }
            }
          >
            BaseApp|Button
          </h1>
          <h2
            style={
              Object {
                "fontSize": "22px",
                "fontWeight": 400,
                "margin": "0 0 10px 0",
                "padding": 0,
              }
            }
          >
            with text
          </h2>
        </div>
        
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Story Source
          </h1>
          <pre
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fafafa",
                "display": "flex",
                "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
                "fontSize": ".88em",
                "justifyContent": "space-between",
                "lineHeight": 1.5,
                "overflowX": "scroll",
                "padding": ".5rem",
              }
            }
          >
            <div>
              <div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;
                    div
                  </span>
                  <span>
                    <span>
                       
                      <span
                        style={Object {}}
                      >
                        style
                      </span>
                      <span>
                        =
                        <span
                          style={Object {}}
                        >
                          {
                          <span
                            style={
                              Object {
                                "color": "#666",
                              }
                            }
                          >
                            {
                            <span>
                              <span
                                style={
                                  Object {
                                    "color": "#666",
                                  }
                                }
                              >
                                textAlign
                              </span>
                            </span>
                            : 
                            <span
                              style={
                                Object {
                                  "color": "#22a",
                                  "wordBreak": "break-word",
                                }
                              }
                            >
                              'center'
                            </span>
                            }
                          </span>
                          }
                        </span>
                      </span>
                      
                    </span>
                  </span>
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &gt;
                  </span>
                </div>
                <div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 33,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &lt;
                      Button
                    </span>
                    <span>
                      <span>
                         
                        <span
                          style={Object {}}
                        >
                          onClick
                        </span>
                        <span>
                          =
                          <span
                            style={Object {}}
                          >
                            {
                            <span
                              style={
                                Object {
                                  "color": "#170",
                                }
                              }
                            >
                              button-click
                            </span>
                            }
                          </span>
                        </span>
                        
                      </span>
                    </span>
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &gt;
                    </span>
                  </div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 48,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      Hello Button
                    </span>
                  </div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 33,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &lt;/
                      Button
                      &gt;
                    </span>
                  </div>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;/
                    div
                    &gt;
                  </span>
                </div>
              </div>
            </div>
            <button
              onClick={[Function]}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "rgb(255, 255, 255)",
                  "borderColor": "rgb(238, 238, 238)",
                  "borderImage": "initial",
                  "borderRadius": "3px",
                  "borderStyle": "solid",
                  "borderWidth": "1px",
                  "cursor": "pointer",
                  "flexShrink": "0",
                  "fontSize": "13px",
                  "overflow": "hidden",
                  "padding": "3px 10px",
                }
              }
              type="button"
            >
              Copy
            </button>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "margin": "20px 0 0 0",
                }
              }
            >
              "
              Button
              " Component
            </h2>
            <table>
              <thead>
                <tr>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    property
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    propType
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    required
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    default
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    description
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    children
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span />
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    yes
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    -
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  />
                </tr>
                <tr>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    onClick
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span />
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    -
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#170",
                        }
                      }
                    >
                      onClick
                    </span>
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  />
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots BaseApp|Button/Emoji with some emoji 1`] = `
<div>
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "border": "1px solid #eee",
          "borderRadius": 3,
          "cursor": "pointer",
          "fontSize": 15,
          "margin": 10,
          "padding": "3px 10px",
        }
      }
      type="button"
    >
      <span
        aria-label="so cool"
        role="img"
      >
        😀 😎 👍 💯
      </span>
    </button>
  </div>
  <button
    onClick={[Function]}
    style={
      Object {
        "background": "#28c",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>
  <div
    style={
      Object {
        "background": "white",
        "bottom": 0,
        "display": "none",
        "left": 0,
        "overflow": "auto",
        "padding": "0 40px",
        "position": "fixed",
        "right": 0,
        "top": 0,
        "zIndex": 99999,
      }
    }
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "#28c",
          "border": "none",
          "borderRadius": "0 0 0 5px",
          "color": "#fff",
          "cursor": "pointer",
          "display": "block",
          "fontFamily": "sans-serif",
          "fontSize": "12px",
          "padding": "5px 15px",
          "position": "fixed",
          "right": 0,
          "top": 0,
        }
      }
      type="button"
    >
      ×
    </button>
    <div>
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "backgroundColor": "#fff",
            "border": "1px solid #eee",
            "borderRadius": "2px",
            "color": "#444",
            "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
            "marginBottom": "20px",
            "marginTop": "20px",
            "padding": "20px 40px 40px",
          }
        }
      >
        <div
          style={
            Object {
              "borderBottom": "1px solid #eee",
              "marginBottom": 10,
              "paddingTop": 10,
            }
          }
        >
          <h1
            style={
              Object {
                "fontSize": "35px",
                "margin": 0,
                "padding": 0,
              }
            }
          >
            BaseApp|Button/Emoji
          </h1>
          <h2
            style={
              Object {
                "fontSize": "22px",
                "fontWeight": 400,
                "margin": "0 0 10px 0",
                "padding": 0,
              }
            }
          >
            with some emoji
          </h2>
        </div>
        <div
          style={
            Object {
              "marginBottom": 0,
            }
          }
        >
          <h3
            id="additional-info"
            style={
              Object {
                "WebkitFontSmoothing": "antialiased",
                "color": "#444",
                "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
                "fontSize": "22px",
                "fontWeight": 600,
                "margin": 0,
                "padding": 0,
                "textTransform": "uppercase",
              }
            }
          >
            Additional Info
          </h3>
          <div
            style={
              Object {
                "WebkitFontSmoothing": "antialiased",
                "color": "#444",
                "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
                "fontSize": "15px",
              }
            }
          >
            You can use markdown to add additional info to a story
          </div>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Story Source
          </h1>
          <pre
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#fafafa",
                "display": "flex",
                "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
                "fontSize": ".88em",
                "justifyContent": "space-between",
                "lineHeight": 1.5,
                "overflowX": "scroll",
                "padding": ".5rem",
              }
            }
          >
            <div>
              <div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;
                    Button
                  </span>
                  <span>
                    <span>
                       
                      <span
                        style={Object {}}
                      >
                        onClick
                      </span>
                      <span>
                        =
                        <span
                          style={Object {}}
                        >
                          {
                          <span
                            style={
                              Object {
                                "color": "#170",
                              }
                            }
                          >
                            button-click
                          </span>
                          }
                        </span>
                      </span>
                      
                    </span>
                  </span>
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &gt;
                  </span>
                </div>
                <div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 33,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &lt;
                      span
                    </span>
                    <span>
                      <span>
                         
                        <span
                          style={Object {}}
                        >
                          role
                        </span>
                        <span>
                          =
                          <span
                            style={Object {}}
                          >
                            "
                            <span
                              style={
                                Object {
                                  "color": "#22a",
                                  "wordBreak": "break-word",
                                }
                              }
                            >
                              img
                            </span>
                            "
                          </span>
                        </span>
                      </span>
                      <span>
                         
                        <span
                          style={Object {}}
                        >
                          aria-label
                        </span>
                        <span>
                          =
                          <span
                            style={Object {}}
                          >
                            "
                            <span
                              style={
                                Object {
                                  "color": "#22a",
                                  "wordBreak": "break-word",
                                }
                              }
                            >
                              so cool
                            </span>
                            "
                          </span>
                        </span>
                        
                      </span>
                    </span>
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &gt;
                    </span>
                  </div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 48,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      😀 😎 👍 💯
                    </span>
                  </div>
                  <div
                    style={
                      Object {
                        "paddingLeft": 33,
                        "paddingRight": 3,
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#444",
                        }
                      }
                    >
                      &lt;/
                      span
                      &gt;
                    </span>
                  </div>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;/
                    Button
                    &gt;
                  </span>
                </div>
              </div>
            </div>
            <button
              onClick={[Function]}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "rgb(255, 255, 255)",
                  "borderColor": "rgb(238, 238, 238)",
                  "borderImage": "initial",
                  "borderRadius": "3px",
                  "borderStyle": "solid",
                  "borderWidth": "1px",
                  "cursor": "pointer",
                  "flexShrink": "0",
                  "fontSize": "13px",
                  "overflow": "hidden",
                  "padding": "3px 10px",
                }
              }
              type="button"
            >
              Copy
            </button>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "margin": "20px 0 0 0",
                }
              }
            >
              "
              Button
              " Component
            </h2>
            <table>
              <thead>
                <tr>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    property
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    propType
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    required
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    default
                  </th>
                  <th
                    style={
                      Object {
                        "textAlign": "left",
                        "verticalAlign": "top",
                      }
                    }
                  >
                    description
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    children
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span />
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    yes
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    -
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  />
                </tr>
                <tr>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    onClick
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span />
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    -
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "color": "#170",
                        }
                      }
                    >
                      onClick
                    </span>
                  </td>
                  <td
                    style={
                      Object {
                        "paddingRight": 10,
                        "verticalAlign": "top",
                      }
                    }
                  />
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
